my_game.py

```diff
@@ -270,7 +269,7 @@ def __int__(self, shot_list, target, level=1, **kwargs):
         arcade.schedule(self.change_dir, CONFIG['UFO_DIR_CHANGE_RATE'])
```

Comment:
It would be nice to not use _arcade.schedule()_ anymore. Can you make a timer based soulution?

Reply:
Ok